libraries/preferences/impl/src/main/kotlin/io/element/android/libraries/preferences/impl/store/DefaultAppPreferencesStore.kt

```diff
@@ -31,7 +32,8 @@ private val Context.dataStore: DataStore<Preferences> by preferencesDataStore(na
 private val developerModeKey = booleanPreferencesKey("developerMode")
 private val customElementCallBaseUrlKey = stringPreferencesKey("elementCallBaseUrl")
 private val themeKey = stringPreferencesKey("theme")
-private val hideImagesAndVideosKey = booleanPreferencesKey("hideImagesAndVideos")
+private val hideInviteAvatarsKey = booleanPreferencesKey("hideInviteAvatars")
+private val timelineMediaPreviewValueKey = stringPreferencesKey("timelineMediaPreviewValue")
```

Comment:
Maybe create a `AppMigration08` class to ensure that if previous key `hideImagesAndVideos` was set to true, `hideInviteAvatars` is set to true and `timelineMediaPreviewValue` is set to `Private`?

Reply:
We discussed this internally and we said to not bother with migration

Reply:
(it's a quick thing to do, but OK)